docs/addon/walkthrough/README.md

```diff
@@ -240,48 +286,106 @@ should see the deployment etc that the operator has created.
 
 e.g. `kubectl get pods -l k8s-app=kubernetes-dashboard`
 
-### Running on-cluster
+## Running on-cluster
 
-Previously we were running on your machine, using your kubernetes credentials.
-For real-world operation, we want to run as a Pod on the cluster, likely launched as part
-of a Deployment.  For that, we'll need a Docker image and some manifests.
+Previously we were running on your machine using your kubernetes credentials.
+We want to run as a Pod on the cluster for real world operator. For that, 
+we'll need a Docker image and some manifests.
 
-We can use bazel to create a docker image and remove the autogenerated image.
+### Building the operator image
 
-```bazel
-rm Dockerfile
-cat > BUILD.bazel <<EOF
-load(
-  "@io_bazel_rules_docker//container:container.bzl",
-  "container_image",
-  "container_push",
-)
+1. Modify the IMG value in the `Makefile` to reflect a docker registry that you 
+   can write to:
 
-container_image(
-  name = "image",
-  base = "//images:kubectl_base",
-  cmd = ["/manager"],
-  directory = "/",
-  files = ["//dashboard-operator/cmd/manager"],
-  tars = ["//dashboard-operator/channels:tar"],
-)
+   ```make
+   # Image URL to use all building/pushing image targets
+   IMG ?= gcr.io/<my-cool-project>/dashboard-operator:latest
+   ```
 
-container_push(
-  name = "push-image",
-  format = "Docker",
-  image = ":image",
-  registry = "{STABLE_DOCKER_REGISTRY}",
-  repository = "{STABLE_PROJECT_NAME}/dashboard-operator",
-  stamp = True,
-  tag = "{STABLE_DOCKER_TAG}",
-)
-EOF
-```
+1. Modify the memory limit for the Operator located in: `config/manager/manager.yaml`:
+
+	```yaml
+	apiVersion: apps/v1
+	kind: StatefulSet
+	spec:
+	  template:
+	     spec:
```

Comment:
Is it missing `containers:` ?
What do you think about below?
```
apiVersion: apps/v1
kind: StatefulSet
spec:
  template:
    spec:
      containers:
      - name: manager
        resources:
          limits:
            cpu: 100m    # unchanged
            memory: 150m # updated
```

Reply:
Or maybe just drop everything but the `resources` block?  Unless you intended it to be a kustomize patch?

Reply:
Thanks to you both for looking a this! `kustomize` patch is a great idea. I'll do that.